x/relayoracle/abci.go

```diff
@@ -0,0 +1,20 @@
+package relayoracle
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/ojo-network/ojo/x/relayoracle/keeper"
+)
+
+// TODO: this might not be necessary, could return the data immediately?
```

Comment:
I like the queueing feature, allows the ibc app to have some control on the amount of requests for each block. This also doesn't change the user experience since the tx wouldn't be processed until the end of the block anyways.